news/7738.feature

```diff
@@ -0,0 +1 @@
+Moved the static metadata into ``setup.cfg``.
```

Comment:
I would put this in a file with a `.trivial` extension as described in [the docs](https://pip.pypa.io/en/stable/development/contributing/#choosing-the-type-of-news-entry) since moving metadata to a static file is not user facing.